src/jsMain/kotlin/component/Node.kt

```diff
@@ -1,16 +0,0 @@
-package component
-
-import model.Node
-import react.FC
-import react.Props
-import react.dom.html.ReactHTML.div
-
-external interface NodeProps : Props {
-    var node: Node
-}
-
-val Node = FC<NodeProps> { props: NodeProps ->
```

Comment:
Why are we deleting this file??

Reply:
We decided that it was simpler to inline the metadata into the Tree itself, rather than give the tree a reference to the Node.

Reply:
Ah! Makes sense

Reply:
(If anyone is reading this, I'm demonstrating git behavior to a friend)